docs/concepts/capabilities.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Capabilities'
+icon: 'ti-anchor' # themify icon pack : https://themify.me/themify-icons
+description: 'Application declaration, deployment, and management'
+sidebar_position: 1
+type: 'docs'
+---
+
+## Overview
+
+A ***capability** is a pluggable piece of functionality for an application. More specifically, capabilities are common, abstracted functionalities&mdash;usually the sort of utilities that aren't core business logic for a given application, such as...
+
+* HTTP Client & Server
+* Logging
+* Key-Value Storage
+* Messaging
+* Blob Storage
+* Secrets
```

Comment:
```suggestion
* Key-Value Storage for caching (think Redis)
* Messaging for persistent message (think Kafka)
* Blob Storage for long term data storage (think S3)
* Secrets for secret management (think Vault)
* HTTP Client for sending HTTP requests
* HTTP Server for receiving HTTP requests
* Logging
```
I like tying these to services that people actually use/think about as an analog -- what do you think?